tools/raisinbread_refresh.php

```diff
@@ -0,0 +1,286 @@
+
+$cwd = getcwd();
+if (substr_compare($cwd, 'tools', mb_strlen($cwd) - mb_strlen('tools')) !== 0) {
+    die('Please run this script from the tools/ directory.' . PHP_EOL);
+}
+
+try {
+    require_once 'generic_includes.php';
+
+    $dbInfo = $config->getSettingFromXML('database');
+
+    if (! $config->getSetting('dev')['sandbox']) {
+        throw new \LorisException(
+            "Config file indicates that this is not a sandbox. Aborting to " .
+            "prevent accidental data loss."
+        );
+    }
+    $dbname = $dbInfo['database'];
+    $host = $dbInfo['host'];
+    // The "quat" user and password should be configured to a user with DROP
+    // and CREATE permissions for the database.
+    $username = $dbInfo['quatUser'];
+    $password = $dbInfo['quatPassword'];
+
+    $urlConfigSetting = $config->getSetting('url');
+    $baseConfigSetting = $config->getSetting('base');
+    $hostConfigSetting = $config->getSetting('host');
+
+} catch (\DatabaseException $e) {
+    echo 'Could not connect to the database in the Config file.' .
+        'It\'s possible that the LORIS tables have already been dropped.' .
+        PHP_EOL;
+    echo 'Continue attempting to install Raisinbread database? (yN)' . PHP_EOL;
+    $input = trim(fgets(STDIN));
+
+    if (mb_strtolower($input) !== 'y') {
+        die;
+    }
+    echo PHP_EOL . 'Please enter the name of your database:' . PHP_EOL;
+    $dbname = trim(fgets(STDIN));
+} catch (\LorisException $e) {
+    die($e->getMessage());
+} catch (Exception $e) {
+    die("Could not load project/config.xml");
+}
+
+
+echo <<<CONFIRMATION
+Please type the database name `$dbname` to confirm you wish to drop tables
+and import test data: 
+CONFIRMATION;
+
+$input = trim(fgets(STDIN));
+if ($input !== $dbname) {
+    die('Input did not match database name. Exiting.');
+}
+
+$mysqlCommand = <<<CMD
+mysql -A $dbname
+CMD;
+
+// Test whether a connection to MySQL is possible via a config file.
+// If not, read DB information from the config file. This method is not as 
+// preferable because it generates MySQL warnings due to the password being
+// supplied via a command-line argument.
+exec($mysqlCommand . ' -e "show tables;" 2>&1 1>/dev/null', $output, $status);
+if ($status != 0) {
+    $mysqlCommand = <<<CMD
+mysql -A "$dbname" -u "$username" -h "$host" -p$password
+CMD;
+}
+
+// Drop tables
+echo PHP_EOL .'Dropping LORIS tables....' . PHP_EOL;
+dropTables();
+
+// Print the names of remaining tables, if any. Some tables may remain if they
+// have been created at a different time during development and are not
+// deleted by the above script. Issues with e.g. foreign keys can be a source
+// of error and so it is useful to know what tables remain.
+exec(
+    "$mysqlCommand -e 'SHOW TABLES;'",
+    $tables,
+    $status
+);
+array_shift($tables); // remove the column name
+if (count($tables) > 0) {
+    echo "\e[33mWARNING: Untracked tables still exist in the database:\e[0m\n";
```

Comment:
I have other code that works as a tools wrapper but it was included in a PR that was never merged.

I'll open a new PR for it because I agree it would be helpful.